libs/judicial-system/message/src/lib/message.ts

```diff
@@ -27,13 +27,13 @@ export enum MessageType {
   DELIVERY_TO_POLICE_INDICTMENT = 'DELIVERY_TO_POLICE_INDICTMENT',
   DELIVERY_TO_POLICE_CASE_FILE = 'DELIVERY_TO_POLICE_CASE_FILE',
   DELIVERY_TO_POLICE_CASE_FILES_RECORD = 'DELIVERY_TO_POLICE_CASE_FILES_RECORD',
-  DELIVERY_TO_POLICE_SUBPOENA_FILE = 'DELIVERY_TO_POLICE_SUBPOENA_FILE', // This delivers the subpoena to LÖKE
+  DELIVERY_TO_NATIONAL_COMMISSIONERS_OFFICE_SUBPOENA = 'DELIVERY_TO_NATIONAL_COMMISSIONERS_OFFICE_SUBPOENA', // This delivers the subpoena to the centralized file service to be served
```

Comment:
Nitpick, I tend to order these events based on who we are delivering to, rather than what we are delivering. Not that it matters which we choose, but it helps to be consistent.